docs/docs/1.1.x/policies/fault-injection.md

```diff
@@ -65,7 +65,9 @@ spec:
 `FaultInjection` is a policy, which is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination dataplanes.
 
 ::: warning
-`FaultInjection` policy available only for L7 HTTP traffic, `kuma.io/protocol: http` is mandatory tag for the destination selector.
+`FaultInjection` policy available only for L7 HTTP traffic, 
```

Comment:
shouldn't be
```suggestion
`FaultInjection` policy is available only for L7 HTTP traffic, 
```
?

Reply:
Right, thanks! Let me backport this for all versions